TeamCode/build.gradle

```diff
@@ -12,6 +12,18 @@
 // Custom definitions may go here
 
 // Include common definitions from above.
+buildscript{
```

Comment:
Great job with this! You did forget all of the other lines of code that you need to make FTC fast load work :) have a great day.